File: unique-room-number/uniqueroomnumber_test.go
package uniqueroomnumber_test

import (
	"testing"

	uniqueroomnumber "github.com/hsmtkk/fuzzy-giggle/unique-room-number"
	"github.com/stretchr/testify/assert"
)

func TestRotatedSame(t *testing.T) {
	assert.True(t, uniqueroomnumber.RotatedSame(86098))
	assert.False(t, uniqueroomnumber.RotatedSame(12345))
}

func Test1(t *testing.T) {
	want := 2
	got := uniqueroomnumber.Solve(1, 9)
	assert.Equal(t, want, got)
}

func Test2(t *testing.T) {
	want := 32
	got := uniqueroomnumber.Solve(100, 10000)
	assert.Equal(t, want, got)
}

File: unique-room-number/unique_room_number_test.py
import unittest

import unique_room_number


class TestUniqueRoomNumber(unittest.TestCase):
    def test1(self):
        want = 2
        got = unique_room_number.solve(1, 9)
        self.assertEqual(want, got)

    def test2(self):
        want = 32
        got = unique_room_number.solve(100, 10000)
        self.assertEqual(want, got)


if __name__ == "__main__":
    unittest.main()
